### DISCUSSION TITLE
验证码处理失败

### DISCUSSION THREAD

**Discussion by Xuyoloa** [IceTiki/ruoli-sign-optimization#136] [Q&A]
作者大大，这个问题是出在哪里呢？

『HTTPConnectionPool(host='*******', port=80): Max retries exceeded with url: / (Caused by NewConnectionError(': Failed to establish a new connection: [Errno -2] Name or service not known'))』] Traceback (most recent call last): File "/var/user/handler.py", line 71, in execute self._execute() File "/var/user/handler.py", line 143, in _execute msg = collection.submitForm() File "/var/user/actions/collection.py", line 436, in submitForm self.getSubmitExtension() File "/var/user/actions/collection.py", line 399, in getSubmitExtension self.host, self.session, self.userInfo['deviceId'], signType="collector")) File "/var/user/liteTools.py", line 525, in handleCaptcha raise Exception(f"验证码处理失败, 错误信息: \n『{error}』") Exception: 验证码处理失败, 错误信息: 『HTTPConnectionPool(host='*********', port=80): Max retries exceeded with url: / (Caused by NewConnectionError(': Failed to establish a new connection: [Errno -2] Name or service not known'))』 [默认备注名](1) > [验证码处理失败, 错误信息: 『HTTPConnectionPool(host='api.gxnzdapple.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError(': Failed to establish a new connection: [Errno -2] Name or service not known'))』] Traceback (most recent call last): File "/var/user/handler.py", line 71, in execute self._execute() File "/var/user/handler.py", line 151, in _execute msg = sign.submitForm() File "/var/user/actions/autoSign.py", line 305, in submitForm self.getSubmitExtension() File "/var/user/actions/autoSign.py", line 267, in getSubmitExtension self.host, self.session, self.userInfo['deviceId'], signType="sign")) File "/var/user/liteTools.py", line 525, in handleCaptcha raise Exception(f"验证码处理失败, 错误信息: \n『{error}』") Exception: 验证码处理失败, 错误信息: 『HTTPConnectionPool(host='******', port=80): Max retries exceeded with url: / (Caused by NewConnectionError(': Failed to establish a new connection: [Errno -2] Name or service not known'))』

**Comment by IceTiki** [comment]
有完整日志吗（一般是txt文件），可以用https://paste.ubuntu.com/或者直接上传文件。

**Comment by Xuyoloa** [comment]
https://paste.ubuntu.com/p/h6bcCgBGpj/

**Comment by IceTiki** [comment]
> https://paste.ubuntu.com/p/h6bcCgBGpj/
> 

你被群友玩弄了，apple是错的

**Reply by beggerlove** [reply]
作者你好   apple是抓什么呀
